src/main/java/com/class4/movieList/mapper/MovieListMapper.java

```diff
@@ -7,24 +7,42 @@
 import com.class4.command.GenreVO;
 import com.class4.command.MovieInfoVO;
 import com.class4.command.MovieListVO;
+import com.class4.command.UserVO;
+import com.class4.command.mapping.MovieActorVO;
+import com.class4.command.mapping.MovieDirectorVO;
+import com.class4.command.mapping.MovieGenreVO;
 import com.class4.movie.util.Criteria;
 
 public interface MovieListMapper {
-	public ArrayList<MovieListVO> cMovieList(Criteria cri);
-	public ArrayList<MovieListVO> eMovieList(Criteria cri);
-	public int getTotalC();
-	public int getTotalE();
-	public int update(MovieInfoVO vo);
-	public ArrayList<String> getGenreList();
-	
-	public void regist(MovieInfoVO vo);
-	public ArrayList<String> getCodeList();
-	public MovieInfoVO getMovieInfo(String cd);
-	public ArrayList<String> getActorCodeList();
-	public ArrayList<String> getDirectorCodeList();
-	public ArrayList<String> getPeopleCodeList();
-	public void ActorInsert(ActorVO vo);
-	public void DirectorInsert(DirectorVO vo);
+   public ArrayList<MovieListVO> cMovieList(Criteria cri);
+   public ArrayList<MovieListVO> eMovieList(Criteria cri);
+   public int getTotalC();
+   public int getTotalE();
+   public void update(MovieInfoVO vo);
+   public ArrayList<String> getGenreList();
+   
+   public void regist(MovieInfoVO vo);
+   public ArrayList<String> getCodeList();
+   public MovieInfoVO getMovieInfo(String cd);
+   public ArrayList<String> getActorCodeList();
+   public ArrayList<String> getDirectorCodeList();
+   public boolean postUpload(MovieInfoVO vo);
 
-	public void GenreInsert(GenreVO vo);
+   public void ActorInsert(ActorVO vo);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.